examples/elevation/components/elevation/elevation.tsx

```diff
@@ -0,0 +1,63 @@
+import {useEffect} from 'react';
+import {
+  useElevationService,
+  useGoogleMap
+} from '@ubilabs/google-maps-react-hooks';
+
+const Elevation = () => {
+  const {map} = useGoogleMap();
+  // Get the elevator from the useElevationService hook
+  const elevator = useElevationService();
+
+  useEffect(() => {
+    if (!map) {
+      return () => {};
+    }
+
+    // Create a new InfoWindow
+    const initialPosition = {lat: 51.08998021141488, lng: 10.627828045134935};
+    const newInfoWindow = new google.maps.InfoWindow({
+      content: 'Click somewhere on the map to see the elevation',
+      position: initialPosition
+    });
+
+    map.setCenter(initialPosition);
+
+    if (!newInfoWindow) {
+      return () => {};
+    }
```

Comment:
Is it possible that `new google.maps.InfoWindow(...)` doesn't return an info window?

```suggestion
```